test/server.test.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-undef*/
+describe('AdobeServerSide Forwarder', function () {
+    var server = new MockHttpServer(),
+        MockVisitorInstance = function() {
+            this.visitorInstance = null;
+            this.orgId = null;
+            this.getInstanceCalled = false;
+            this.userId = null;
+
+            this.setCustomerIDs = function(userIdObject) {
+                this.userId = userIdObject;
+            };
+            this.getCustomerIDs = function() {
+                return this.userId;
+            };
+        },
+
+        Visitor = {
+            getInstance: function(orgId) {
+                var instance = new MockVisitorInstance;
+                instance.orgId = orgId;
+                instance.getInstanceCalled = true;
+                instance.getMarketingCloudVisitorID = function(cb) {
+                    cb('MCID test');
+                };
+                return instance;
+            }
+        };
+
+    beforeAll(function () {
+        window.Visitor = Visitor;
+        server.start();
+    });
+
+    beforeEach(function() {
+        window.s = null;
+        window.mockInstances = {};
+        window.mParticleAndroid = null;
+        window.mParticle.isIOS = null;
+        window.mParticle.useCookieStorage = false;
+        mParticle.isDevelopmentMode = false;
+        mParticle.config = {
+            requestConfig: false,
+            workspaceToken: 'testworkspacetoken',
+            kitConfigs: [
+                {
+                    name: 'Adobe',
+                    settings: {
```

Comment:
Nitpick: I'd rather load this kit config from a json file or external js to keep the test readable.